admin_interface/locale/tr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,208 @@
+# This file is distributed under the same license as the
+# django-admin-interface package.
+#
+# Translators:
+# Martin Vuelta <martin.vuelta@softbutterfly.io>, 2018
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: django-admin-interface\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-09-04 09:18+0000\n"
+"PO-Revision-Date: 2020-01-24 06:07+0300\n"
+"Last-Translator: Martin Vuelta <martin.vuelta@softbutterfly.io>\n"
+"Language-Team: Spanish <dev@softbutterfly.io>\n"
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
Since you duplicated an existing translation, is this `Plural-Form` correct for TR localization?

Reply:
It's my fault. I fixed it on new commit.